### DISCUSSION TITLE
LAB3 fir-tb problems

### DISCUSSION THREAD

**Discussion by s095339** [bol-edu/HLS-SOC-Discussions#89] [General]
大家好，小弟在使用makefile跑tb的時候，有發現一些問題
tb的一些訊號可能有誤
如下，綠色的部分是修改後的結果，tap_ram的訊號出現了兩次，應改為data_ram
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/78518884/a54b03ae-147f-4c4c-a5ba-255e3693686e)

然後是他抓不到bram的問題，可能要改./fir/include.rtl.list.xsim的內容才抓的到
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/78518884/f32c72b1-1682-4000-9d13-d62f45556b3a)

最後就是bram12.v 少大括弧
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/78518884/d539dbde-9c23-49fa-acbc-3093a617bf5e)

**Comment by ZheChen-Bill** [comment]
同學你好，已經針對你提出的第一個問題和第三個問題進行修正了，也請其他有遇到此問題的同學參考。
